docs/src/pages/components/data-grid/overview/overview.md

```diff
@@ -11,10 +11,10 @@ The component leverages the power of React and TypeScript, to provide the best U
 
 ## Overview
 
-Data tables display information in a grid-like format of rows and columns. They organize information in a way that’s easy to scan so that users can look for patterns and insights. The data grid comes in 2 versions:
+Data tables display information in a grid-like format of rows and columns. They organize information in a way that’s easy to scan so that users can look for patterns and insights. The data grid comes in 2 versions, both in Material-UI X:
 
-- `DataGrid` **MIT licensed** as part of the community edition. It's an extension of `@material-ui/core`.
-- `XGrid` **Commercially licensed** as part of the X product line offering.
+- `DataGrid` **MIT licensed** as part of the Community plan.
+- `XGrid` **Commercially licensed** as part of Pro and Premium plans.
```

Comment:
Maybe we can add something like
DataGrid is a subset of XGrid therefore all the demos and docs about DataGrid are also applicable for XGrid

Reply:
We could: #1580, but it's not the problem I'm going after.

The problem I'm going after is to solve the confusion between Material-UI core and Material-UI X. The former is fully MIT, the later is based on an open core business model.